src/aind_metadata_service/client.py

```diff
@@ -23,19 +24,43 @@ class StatusCodes(Enum):
 class AindMetadataServiceClient:
     """Class to handle client api calls to the service."""
 
-    def __init__(self, domain: str):
+    PROD_URL = "http://aind-metadata-service"
+    DEV_URL = "http://aind-metadata-service-dev"
```

Comment:
Most clients require the user to set the domain (and other credentials) instead of hard-coding them in code.

Reply:
I'd argue that since most users will be internal that the priority should be ease of use for those users. This pattern avoids the need for users to remember and type the two URLs. It also makes it easy to change the URLs in one place if they need to change in the future (e.g. if we want to start using https instead of http).

For improved modularity, I moved all constants to a new constants.py. Is that an improvement?